xcode/Subconscious/Shared/Components/Common/Profile/FollowUserSheet.swift

```diff
@@ -169,15 +169,14 @@ struct FollowUserSheetFormCursor: CursorProtocol {
 }
 
 struct FollowUserSheet: View {
-    var state: FollowUserSheetModel
-    var send: (FollowUserSheetAction) -> Void
+    var store: ViewStore<FollowUserSheetModel>
+   
+    var state: FollowUserSheetModel { store.state }
+    var send: (FollowUserSheetAction) -> Void { store.send }
     
     var onAttemptFollow: () -> Void
```

Comment:
Now that we're bringing a viewstore into this view, we should not need callbacks, since we should handle via view store update function and `ViewStore.send()`